server/routes/appStatus.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @file Defines all routes for the App Status route.
+ */
+
+const express = require('express');
+const { createInitialStatus, retrieveAppStatus } = require('../db/queries');
+const { asyncWrapper } = require('../middleware');
+const plaid = require('../plaid');
+
+const router = express.Router();
+const { PLAID_CLIENT_ID, PLAID_SECRET_SANDBOX } = process.env;
+
+/**
+ * Sets up initial app status if it doesn't already exist.
+ *
+ * @returns {Object[]} an array of app funds
+ */
+router.get(
+  '/initial',
+  asyncWrapper(async (req, res) => {
+    try {
+      const status = await retrieveAppStatus();
+
+      if (status.length === 0) {
+        // get the last event_id
+        const transferEventListRequest = {
+          client_id: PLAID_CLIENT_ID,
+          secret: PLAID_SECRET_SANDBOX,
+        };
+        const response = await plaid.transferEventList(
+          transferEventListRequest
+        );
+
+        await createInitialStatus(response.data.transfer_events[0].event_id);
```

Comment:
Just double checking here since the comment says "last" `event_id`. Is the last `event_id` in this case the 0th index of `response.data.transfer_events` rather than the last index?